### PR TITLE
Add proj_coordoperation_create_inverse to C API

### PR DESCRIPTION
**Pull request by nyalldawson**
…for creating the inverse operation given a coordinate operation

### CODE REVIEW

File: src/iso19111/c_api.cpp [hunk 1]
```diff
@@ -7769,6 +7769,33 @@ PJ *proj_normalize_for_visualization(PJ_CONTEXT *ctx, const PJ *obj) {
 
 // ---------------------------------------------------------------------------
 
+/** \brief Returns a PJ* coordinate operation object which represents the
+ * inverse operation of the specified coordinate operation.
+ *
+ * @param ctx PROJ context, or NULL for default context
+ * @param obj Object of type CoordinateOperation (must not be NULL)
+ * @return a new PJ* object to free with proj_destroy() in case of success, or
+ * nullptr in case of error
```
Comment (rouault) [inline src/iso19111/c_api.cpp:7778]:
```
@since 6.3
```

File: src/proj.h [hunk 2]
```diff
@@ -363,6 +363,7 @@ PJ PROJ_DLL *proj_create_crs_to_crs_from_pj(PJ_CONTEXT *ctx,
                                             PJ_AREA *area,
                                             const char* const *options);
 PJ PROJ_DLL *proj_normalize_for_visualization(PJ_CONTEXT *ctx, const PJ* obj);
+PJ PROJ_DLL *proj_coordoperation_create_inverse(PJ_CONTEXT *ctx, const PJ *obj);
```
Comment (rouault) [inline src/proj.h]: could you move that down to to the other proj_coordoperation_ declarations ?

File: test/unit/test_c_api.cpp [hunk 3]
```diff
@@ -3486,6 +3486,36 @@ TEST_F(CApi, proj_normalize_for_visualization_on_crs) {
...
+                "+proj=unitconvert +xy_in=deg +xy_out=rad +step +proj=push "
+                "+v_3 +step +proj=cart +ellps=evrst30 +step +proj=helmert "
+                "+x=293 +y=836 +z=318 +rx=0.5 +ry=1.6 +rz=-2.8 +s=2.1 "
+                "+convention=position_vector +step +inv +proj=cart "
+                "+ellps=WGS84 +step +proj=pop +v_3 +step +proj=unitconvert "
+                "+xy_in=rad +xy_out=deg +step +proj=axisswap +order=2,1");
+    ObjectKeeper keeper_P(P);
+    ASSERT_NE(P, nullptr);
+    auto Pinversed = proj_coordoperation_create_inverse(m_ctxt, P);
+    ObjectKeeper keeper_Pnormalized(Pinversed);
```
Comment (rouault) [inline test/unit/test_c_api.cpp]:
```suggestion
    ObjectKeeper keeper_Pinversed(Pinversed);
```

**Comment by nyalldawson** [comment]
@rouault all done!

**Comment by nyalldawson** [comment]
Thanks @rouault!

**Pull request merged**